addons/intel/e810.go

```diff
@@ -89,15 +90,21 @@ func OnPTPConfigChangeE810(data *interface{}, nodeProfile *ptpv1.PtpProfile) err
 				glog.Error("e810 failed to unmarshal opts: " + err.Error())
 			}
 			if e810Opts.EnableDefaultConfig {
-				stdout, err = exec.Command("/usr/bin/bash", "-c", EnableE810PTPConfig).Output()
+				stdout, _ = exec.Command("/usr/bin/bash", "-c", EnableE810PTPConfig).Output()
 				glog.Infof(string(stdout))
 			}
 			if (*nodeProfile).PtpSettings == nil {
 				(*nodeProfile).PtpSettings = make(map[string]string)
 			}
 			for device, pins := range e810Opts.DevicePins {
 				dpllClockIdStr := fmt.Sprintf("%s[%s]", dpll.ClockIdStr, device)
-				(*nodeProfile).PtpSettings[dpllClockIdStr] = strconv.FormatUint(getClockIdE810(device), 10)
+				if _, found := (*nodeProfile).PtpSettings["unitTest"]; found {
```

Comment:
This is only relevant when running unit tests?

Reply:
Can you include a comment in the plugin explaining that?

Reply:
Done

Reply:
Or did you mean in the README?